src/Operation/Transpose.php

```diff
@@ -26,46 +26,45 @@ final class Transpose extends AbstractOperation
     /**
      * @pure
      *
+     * @psalm-suppress ImpureMethodCall
```

Comment:
if we do end up keeping this we should always have clarifying comments for them, e.g.

```
@psalm-suppress ImpureMethodCall - using MultipleIterator as an internal tool which is not returned

Reply:
I just see this message! Good point, going to make the change.